org.gnome.Loupe.yml

```
@@ -0,0 +1,128 @@
+id: org.gnome.Loupe
+runtime: org.gnome.Platform
+runtime-version: '44'
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-stable
+  - org.freedesktop.Sdk.Extension.llvm15
+command: loupe
+finish-args:
+  # Graphics
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  # Browse through photos
+  - --filesystem=host
+  # Enable access to GVfs
+  - --talk-name=org.gtk.vfs.*
+  - --filesystem=xdg-run/gvfs
+  - --filesystem=xdg-run/gvfsd
+build-options:
+  append-path: /usr/lib/sdk/rust-stable/bin:/usr/lib/sdk/llvm15/bin
+cleanup:
+  - /include
+  - /lib/pkgconfig
+  - /man
+  - /share/doc
+  - /share/gtk-doc
+  - /share/man
+  - /share/pkgconfig
+  - '*.la'
+  - '*.a'
+add-extensions:
+  # Exile HEIC (x265) codec into extension
+  org.gnome.Loupe.Devel.HEIC:
```

```suggestion
  org.gnome.Loupe.HEIC:
```

Or maybe we want `org.gnome.Loupe.Decoder.HEIC`?

---

Thanks!